src/Query/Ordening.php

```diff
@@ -0,0 +1,66 @@
+<?php
+declare(strict_types = 1);
+
+namespace TBolier\RethinkQL\Query;
+
+use TBolier\RethinkQL\Message\MessageInterface;
+use TBolier\RethinkQL\RethinkInterface;
+use TBolier\RethinkQL\Types\Term\TermType;
+
+class Ordening extends AbstractQuery implements OrdeningInterface
```

Comment:
Is any operation possible after ordering? Should it really extend AbstractQuery?

Reply:
Yes, more operations are possible.